app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/coverage/PerilBottomSheet.kt

```diff
@@ -2,68 +2,204 @@ package com.hedvig.app.feature.insurance.ui.detail.coverage
+
+            dialog?.setOnShowListener { dialogInterface ->
+                val containerLayout =
+                    (dialogInterface as BottomSheetDialog).findViewById<FrameLayout>(com.google.android.material.R.id.container)
+                val shadow =
+                    bottomSheetDialog.layoutInflater.inflate(R.layout.bottom_sheet_shadow, null)
+                val chevron = shadow.findViewById<ImageView>(R.id.chevron)
+                chevron.setHapticClickListener { expandSheet() }
+
+                shadow.layoutParams = FrameLayout.LayoutParams(
+                    FrameLayout.LayoutParams.MATCH_PARENT,
+                    FrameLayout.LayoutParams.WRAP_CONTENT
+                ).apply {
+                    gravity = Gravity.BOTTOM
+                }
+                containerLayout?.addView(shadow)
+                (dialog as? BottomSheetDialog)?.behavior?.let { behaviour ->
+                    behaviour.addBottomSheetCallback(
+                        object : BottomSheetCallback() {
+
+                            override fun onStateChanged(bottomSheet: View, newState: Int) {
+                                when (newState) {
+                                    BottomSheetBehavior.STATE_EXPANDED -> {
+                                        dialog?.window?.clearFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND)
+                                        dialog?.window?.statusBarColor =
+                                            requireContext().colorAttr(R.attr.colorSurface)
+                                        if (!requireContext().isDarkThemeActive) {
+                                            dialog?.window?.decorView?.let {
+                                                it.systemUiVisibility =
+                                                    it.systemUiVisibility or View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
+                                            }
+                                        }
+
+                                        close.setHapticClickListener {
+                                            this@PerilBottomSheet.dismiss()
+                                        }
+                                    }
+                                    STATE_DRAGGING -> {
+                                        defaultStatusBarColor?.let {
+                                            dialog?.window?.statusBarColor = it
+                                        }
+                                        defaultSystemUiVisibility?.let {
+                                            dialog?.window?.decorView?.systemUiVisibility = it
+                                        }
+                                        dialog?.window?.addFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND)
+                                    }
+                                    BottomSheetBehavior.STATE_COLLAPSED -> {
+                                        close.setOnClickListener(null)
```

Comment:
Is this required?

Reply:
Nope 🧹

Reply:
I think you may have resolved this one too early, it's not removed yet 😬

Reply:
This is so weird, why does this part not get added to my commit?

Reply:
hmm, it is added this commend just does not get the outdated badge for some reason

Reply:
Looks like it's still not added in the Github UI, atleast.